browser-only.js

```diff
@@ -28,6 +28,9 @@ if (typeof window !== 'undefined') {
   // Polyfill window.matchMedia (primarily for IE9)
   require('matchmedia-polyfill');
   require('matchmedia-polyfill/matchMedia.addListener');
+
+  // Polyfill CustomEvent
+  require('custom-event-polyfill');
```

Comment:
@ljharb I think I got fooled by SEO, was not aware of this. I'll update accordingly. Thanks for catching this.